src/components/test/Calculator.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
- [ ] As requested by the previous Code reviewer, make sure all the testes are passing 😄 , most of them are passing but not all 👍 
![Capture](https://user-images.githubusercontent.com/80458199/230503303-b5c62619-1844-471b-910a-ecad16a2c21f.PNG)

Reply:
hello i think what you need to do is install the react-test-render

Reply:
can i have a meeting with you please

Reply:
coz i see all the project are passing on my end, even on my partners end